README.md

```diff
@@ -16,16 +16,14 @@ git clone https://github.com/Tendrl/tendrl-ansible.git
 ## What does it do?
 
 There are ansible roles for installation of [Tendrl](http://tendrl.org/), based
-on [upstream documentation](https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference):
+on [upstream documentation](https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference(Revised)):
```

Comment:
Hey, could you change it to 

"https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference"

The "Revised" part was temporary since we had to support older install doc